src/main/java/io/litmuschaos/LitmusClient.java

```diff
@@ -28,23 +28,21 @@ public void close() throws Exception {
     }
 
     // TODO - @Suyeon Jung : host, port config to LitmusAuthConfig class
-    public LoginResponse authenticate(String host, String username, String password) throws IOException {
+    public LoginResponse authenticate(String username, String password) throws IOException {
         LoginRequest request = new LoginRequest(username, password);
-        LoginResponse response = httpClient.post(host + "/login", request, LoginResponse.class);
+        LoginResponse response = httpClient.post("/login", request, LoginResponse.class);
         this.token = response.getAccessToken();
         return response;
```

Comment:
In litmusClient constructor, we initialize token value by `credential.getAccessToken()` method. 
So, I think initializing part in `authenticate` method is not required. If authenticate method is used by client independently, I think removing initialize part in constructor is better solution.

Reply:
Hi @jemlog
Thanks for the review. As authenticate method should served independtly, I've removed duplicated token initialization in `LitmusClient` constructor. 

```java
public LitmusClient(String host, String username, String password) throws IOException {
        this.httpClient = new LitmusHttpClient(host);
        this.authenticate(username, password);
    }
```

```java
public LoginResponse authenticate(String username, String password) throws IOException {
        LoginRequest request = new LoginRequest(username, password);
        LoginResponse response = httpClient.post("/login", request, LoginResponse.class);
        this.token = response.getAccessToken();
        return response;
    }
```